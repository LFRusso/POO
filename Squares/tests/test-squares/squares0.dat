s1 -3 0.5 2
s2 -2 2 3
s3 -1.5 3.5 1
s4 1 1 1.5
s5 0 -2 1.5
