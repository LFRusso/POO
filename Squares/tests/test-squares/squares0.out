s1 intercepts s2 with area 0.5
s2 intercepts s3 with area 1
